lib/splitclient-rb/engine/api/events.rb

```diff
@@ -22,10 +22,12 @@ def post
             'SplitSDKVersion' => event_slice[0][:m][:s]
           )
 
-          if (200..299).include? result.status
-            SplitIoClient.configuration.logger.debug("Events reported: #{event_slice.size}") if SplitIoClient.configuration.debug_enabled
+          if response.success?
```

Comment:
Does this .sucess? match any status code >=200 and < 300? Out events BE doesn't return 200 exactly, but sometimes 203, 204, etc

Reply:
The check for the range is included on Faraday. Already discussed on site.